### PR TITLE
update keras examples

### PR DESCRIPTION
**Pull request by kshitij12345**
# Description

Include a summary of the changes and the related issue.

__Related to:__ `(ClickUp/JIRA task name)`

---

## ❔ This change

- [ ] adds a new feature
- [ ] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

Mark which items have been completed:

- [ ] Refactored code ([sourcery](https://sourcery.ai/))
- [ ] Tested code locally
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)

### 🪝 Run the following to both [pre-commit](https://pre-commit.com/) hooks and JupyterLab (turn on `format on save`)

- [ ] Stripped outputs from notebooks - ([nbstripout](https://pypi.org/project/nbstripout/))
- [ ] Sorted imports - ([isort](https://pycqa.github.io/isort/))
- [ ] Formatted code - ([back](https://github.com/psf/black))

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

File: integrations-and-supported-tools/keras/scripts/Neptune_Keras_more_options.py [hunk 1]
```diff
@@ -22,15 +22,15 @@
...
 run["hyper-parameters"] = params
 
 optimizer = tf.keras.optimizers.SGD(learning_rate=params["lr"], momentum=params["momentum"])
 
 model.compile(optimizer=optimizer, loss="sparse_categorical_crossentropy", metrics=["accuracy"])
 
-# log metrics during training
-neptune_cbk = NeptuneCallback(run=run, base_namespace="metrics")
+# (Neptune) log metrics during training
+neptune_cbk = NeptuneCallback(run=run)
```
Comment (kshitij12345) [inline integrations-and-supported-tools/keras/scripts/Neptune_Keras_more_options.py:33]: Latest version doesn't require passing `base_namespace` and defaults to training.

File: integrations-and-supported-tools/keras/scripts/Neptune_Keras_more_options.py [hunk 2]
```diff
@@ -39,13 +39,15 @@
     callbacks=[neptune_cbk],
 )
 
-# log images
-for image in x_test[:100]:
-    run["test/sample_images"].log(File.as_image(image))
+# (Neptune) log test images with prediction
+for image, label in zip(x_test[:10], y_test[:10]):
```
Comment (kshitij12345) [inline integrations-and-supported-tools/keras/scripts/Neptune_Keras_more_options.py:43]: Logged fewer images but with predicted and expected label.

File: integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb [hunk 3]
```diff
@@ -23,7 +23,9 @@
     "* Initialize Neptune and create a `run`,\n",
     "* Create a `NeptuneCallback()`,\n",
     "* Log model training metrics to Neptune using `NeptuneCallback()`\n",
-    "* Log additional metadata like hyperaparameters, images, and model weights to Neptune."
+    "* Log additional metadata like hyperaparameters, images, and model weights to Neptune.\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/tf-keras-integration/e/TFK-40889/all)"
```
Comment (kshitij12345) [inline integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb:28]: @SiddhantSadangi I can't add showcase run tag to this run as I don't have the necessary rights. Please add the tag. Thanks!

File: README.md [hunk 4]
```diff
@@ -88,6 +88,7 @@ You can run every example with zero setup as an "ANONYMOUS" Neptune user (no reg
...
 | fastai | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/fastai) | [![neptune]](https://app.neptune.ai/o/common/org/fastai-integration/e/FAS-61/dashboard/fastai-dashboard-1f456716-f509-4432-b8b3-a7f5242703b6) | [![github]](https://github.com/neptune-ai/examples/tree/main/integrations-and-supported-tools/fastai/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/main/integrations-and-supported-tools/fastai/notebooks/Neptune_fastai.ipynb) |
+| Keras | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/keras) | [![neptune]](https://app.neptune.ai/o/common/org/tf-keras-integration/e/TFK-18/all) | [![github]](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/keras/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb) |
```
Comment (SiddhantSadangi) [inline README.md:91]:
```suggestion
| Keras | [![docs]](https://docs.neptune.ai/integrations-and-supported-tools/model-training/keras) | [![neptune]](https://app.neptune.ai/o/common/org/tf-keras-integration/e/TFK-40889/all) | [![github]](https://github.com/neptune-ai/examples/blob/main/integrations-and-supported-tools/keras/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb) |
```

File: integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb [hunk 5]
```diff
@@ -462,7 +462,7 @@
     ]
    },
    "source": [
-    "## Explore results in the Neptune app"
+    "## Explore results in the Neptune"
```
Comment (SiddhantSadangi) [inline integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb:465]:
```suggestion
    "## Explore results in the Neptune app"
```

File: integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb [hunk 6]
```diff
@@ -355,7 +357,7 @@
     "\n",
     "model.compile(optimizer=optimizer, loss=\"sparse_categorical_crossentropy\", metrics=[\"accuracy\"])\n",
     "\n",
-    "neptune_cbk_2 = NeptuneCallback(run=run_2, base_namespace=\"metrics\")\n",
+    "neptune_cbk_2 = NeptuneCallback(run=run_2)\n",
```
Comment (SiddhantSadangi) [inline integrations-and-supported-tools/keras/notebooks/Neptune_Keras.ipynb:360]:
```suggestion
    "neptune_cbk_2 = NeptuneCallback(run=run_2) # base_namespace defaults to \"training\"\n",
```
Let's add this comment here as we are removing the parameter but mentioning it in the header

File: integrations-and-supported-tools/keras/scripts/run_examples.sh [hunk 7]
```diff
@@ -0,0 +1,8 @@
+echo "Installing requirements..."
```
Comment (SiddhantSadangi) [inline integrations-and-supported-tools/keras/scripts/run_examples.sh:1]:
```suggestion
set -e

echo "Installing requirements..."
```

**Review by SiddhantSadangi — DISMISSED** [review]

**Comment by SiddhantSadangi** [comment]
@Patrycja-J - for visibility

**Review by SiddhantSadangi — DISMISSED** [review]

**Review by normandy7 — DISMISSED** [review]

**Review by SiddhantSadangi — APPROVED** [review]

**Review by normandy7 — APPROVED** [review]
